apps/speed.c

```diff
@@ -1040,33 +1038,31 @@ static int ECDSA_verify_loop(void *args)
 /* ******************************************************************** */
 static long ecdh_c[EC_NUM][1];
 
-static int ECDH_compute_key_loop(void *args)
+static void ECDH_EVP_derive_key(unsigned char *derived_secret,
```

Comment:
Actually it does not, I was trying to mimic the structure of the old code as close as possible, do you think it would be better to just squeeze the code in just one function?